cloud_Azure/terraform/module/roles.tf

```diff
@@ -1,13 +1,17 @@
-# Provide service principal Contributor role to storage account
+# Provide service principal Contributor role to each storage account
 resource "azurerm_role_assignment" "kentic_role_contributor" {
```

Comment:
typo: kentic

Reply:
Fixed on parent branch, will be merged once approved